### DISCUSSION TITLE
unable to access the student developer pack

### DISCUSSION THREAD

**Discussion by Daipayan-Das** [github/india#439] [Students]
Dear Sir/Mam,
 
Its been many days that I have requested for the student developer pack in the FRT portal, I have updated my LinkedIn profile many days ago and also mentioned Microsoft Azure in the skills and endorsement section but still I did not got any reply and so cannot claim the student developer pack.
 
I the portal it is mentioned that it may take 3days to verify the profile but its been 1 week since I have requested to claim but still I haven’t received it.
 
Please kindly look into the matter and do reply me with an update.

**Comment by PRIYA-S-102** [comment]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
